Core/Core/Store/Store.swift

```diff
@@ -179,6 +180,13 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
         }
     }
 
+    @objc
+    public func controllerWillChangeContent(_ controller: NSFetchedResultsController<NSFetchRequestResult>) {
+        if #available(iOSApplicationExtension 13.0, *) {
+            objectWillChange.send()
+        }
+    }
```

Comment:
if you move this down into the extension, you shouldn't need the second `if #available`

Reply:
Unfortunately extensions and `@objc` don't get along

Reply:
oh, right.

Reply:
maybe an `@available` on the func then?